components/compliance-service/reporting/relaxting/es_types.go

```diff
@@ -98,6 +100,14 @@ type ESInSpecReportControlsResult struct {
 	SkipMessage string  `json:"skip_message,omitempty"`
 }
 
+type ESInSpecReportControlsWaiverData struct {
+	ExpirationDate     string `json:"expiration_date"`
```

Comment:
should this be time.Time instead of string type?

Reply:
discussed in standup that we should leave as string -- this is metadata about the waived control that comes in on the report, no need to query it later